pkgs/by-name/pr/prismlauncher-unwrapped/package.nix

```diff
@@ -109,7 +109,43 @@ stdenv.mkDerivation (finalAttrs: {
     '';
     homepage = "https://prismlauncher.org/";
     changelog = "https://github.com/PrismLauncher/PrismLauncher/releases/tag/${finalAttrs.version}";
-    license = lib.licenses.gpl3Only;
+    # License list sources:
+    # https://github.com/PrismLauncher/PrismLauncher/tree/develop/libraries
+    # https://github.com/PrismLauncher/PrismLauncher/blob/develop/COPYING.md
+    license = with lib.licenses; [
+      # Prism Launcher's license
+      gpl3Only
+      # Original MultiMC license
+      asl20
+      # Prism Launcher assets
+      cc-by-sa-40
+      # LibNBT++, Oxygen Icons
+      lgpl3Plus
+      # rainbow
+      lgpl2Plus
+      # libqrencode, mingw
+      lgpl21Plus
+      # QT, qdcss
+      lgpl3Only
+      # Breeze Icons
+      gpl3Plus
+      # Material Design Icons
+      ofl
+      # lionshead, toml++, vcpkg, qr-code-generator
```

Comment:
Everything after lionshead is dynamically linked to or non existent in the current version of Prism